.github/workflows/release.yml.disabled

```diff
@@ -22,6 +22,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      # https://github.blog/2022-04-12-git-security-vulnerability-announced/
+      - name: Pacify git's permission check
+        run: git config --global --add safe.directory /__w/cockpit-session-recording/cockpit-session-recording
```

Comment:
Woops sorry for the oversight. I fixed this as you proposed though I have not tested it, if it doesn't work it may need
~~~
        run: git config --global --add safe.directory *
~~~

Reply:
It works the way you pushed it now, I use the [same approach in umockdev](https://github.com/martinpitt/umockdev/blob/main/.github/workflows/release.yml#L19). We can apply this to all our projects.